projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/DataJobInitContainerOOMIT.java

```diff
@@ -40,7 +47,23 @@ public void testDataJob_causesOOM_shouldCompleteWithUserError(
     String executionId = executeDataJobResult.getRight();
 
     // Check the data job execution status
-    JobExecutionUtil.checkDataJobExecutionStatus(
+    testDataJobExecutionRead(
```

Comment:
call most of the checks explicitly instead of using the helper method we were using previously so I can skip the logs exist assertion.